concreep-refilled/control.lua

```diff
@@ -230,7 +235,7 @@ function _reinit_reentrant_level2_body()
 		end
 		-- if we got here:
 		--  - we either exhausted the list of roboports (so return true) or
-		--  - we reached ROBOPORTS_PER_REINIT_PASS (so return false)
+		--  - we reached roboports per reinit pass (so return false)

```

Comment:
reference that it's `settings.global["roboports per reinit pass"]`

Reply:
done